puppet/preprovision.sh

```diff
@@ -38,7 +38,6 @@ if [[ ! -f /etc/chassis-updated ]]; then
 	# Update apt
 	sudo apt-get update
 
-	# Install/Upgrade Puppet
```

Comment:
Why remove this line?

Reply:
this was brought in when I merged the main branch into this so I don't know why this was removed, it weren't me!

Reply:
Because I am a stupidhead 😂 I probably just missed it when I was adding it back when Puppet v7 didn't play nicely!